processes/security_release_process.md

```diff
@@ -0,0 +1,80 @@
+# Security Release Process
+
+The security release process covers the steps required to plan/implement
+a security release.
+
+The steps include:
+
+* Get agreement on the list of vulnerabilities to be addressed
+  and the planned date for the releases. This is done in an issue
+  in the private security repo titled `Next Security Release`
+
+* Once agreement on the list and date has been agreed, validate
+  that all vulnerabilities have been assigned a CVE following
+  the [cve_management_process](https://github.com/nodejs/security-wg/blob/master/processes/cve_management_process.md).
+
+* Co-ordinate with the Release team members to line up one
+  or more releasers to do the releases on the agreed date.
+
+* Prep for the pre-security announcement and final security
+  annoucement by getting agreement on drafts following the
+  [security_announcement_process](https://github.com/nodejs/security-wg/blob/master/processes/security_annoucement_process.md). 
+
+* One week in advance of the agreed date for the security
+  release, ensure the pre-announce is sent out as outlined in the
+  [security_announcement_process](https://github.com/nodejs/security-wg/blob/master/processes/security_annoucement_process.md). 
+
+* One week in advance send an email to the docker official image
+  [maintainers](https://github.com/docker-library/official-images/blob/master/MAINTAINERS)
+  with an FYI that security releases will be going out on the agreed date.
+
+* Open an issue in the [docker-node](https://github.com/nodejs/docker-node)
+  repo and get one or more volunteers to be available to review the PR
+  to update Node.js versions in the docker-node repo immediately after the release.
+
+* On the day of the release co-ordinate with the Release
+  team members and keep up to date on progress. Get an guesstimate of
+  when releases may be ready and send an FYI to the docker offical image
+  [maintainers](https://github.com/docker-library/official-images/blob/master/MAINTAINERS).
+
+* When the releases are promoted, ensure the final announce
+  goes out as per the
+  [security_announcement_process](https://github.com/nodejs/security-wg/blob/master/processes/security_annoucement_process.md). 
+
+* Create a PR to update the Node.js version in the official docker images.  
+  * Checkout the docker-node repo
+  * Run the update.sh script to update versions
+  * Update the changed files to **remove** all changes EXCEPT those which 
```

Comment:
@chorrell is working on an update to the script allowing you to pass `-s` which should make sure to _just_ update the node version.

https://github.com/nodejs/docker-node/pull/784

Reply:
@SimenB that's great I was thinking we wanted something to make that easier.

Reply:
And it's landed, so this part should probably be updated

Reply:
updated.